tools/pythonpkg/src/include/duckdb_python/arrow_array_stream.hpp

```diff
@@ -21,6 +21,32 @@
 #include <vector>
 
 namespace duckdb {
+
+namespace pyarrow {
+class RecordBatchReader : public py::object {
+public:
+	RecordBatchReader(const py::object &o) : py::object(o, borrowed_t {}) {
+	}
+	using py::object::object;
+
+public:
+	static bool check_(const py::handle &object) {
+		return !py::none().is(object);
+	}
+};
+class Table : public py::object {
+public:
+	Table(const py::object &o) : py::object(o, borrowed_t {}) {
+	}
+	using py::object::object;
+
+public:
+	static bool check_(const py::handle &object) {
```

Comment:
methods shouldn't finish with _

Reply:
This name is from pybind11, I can't change it